mcux/hal_nxp.cmake

```diff
@@ -46,6 +46,7 @@ include(driver_common)
 
 #Include system_xxx file
 #This can be extended to other SoC series if needed
+if (DEFINED CONFIG_PLATFORM_SPECIFIC_INIT OR DEFINED CONFIG_SOC_SERIES_IMX_RT6XX)
```

Comment:
Can `CONFIG_PLATFORM_SPECIFIC_INIT` also be enabled for `CONFIG_SOC_SERIES_IMX_RT6XX`

Reply:
Yes, it will be with the merge of https://github.com/zephyrproject-rtos/zephyr/pull/52730. However, that PR is currently blocked. I could also update the Zephyr PR associated with this HAL PR: https://github.com/zephyrproject-rtos/zephyr/pull/53675 to enable `PLATFORM_SPECIFIC_INIT` for the RT6xx, but then we would have to add a stub `z_arm_platform_init` for that platform until https://github.com/zephyrproject-rtos/zephyr/pull/52730 merges

What occurred to get to the point that this PR was necessary was the following:
1. https://github.com/zephyrproject-rtos/zephyr/pull/52730 was opened, with HAL PR https://github.com/zephyrproject-rtos/hal_nxp/pull/211 as the associated PR. Since https://github.com/zephyrproject-rtos/zephyr/pull/52730 enabled `CONFIG_PLATFORM_SPECIFIC_INIT` for the RT6xx, that platform built correctly and passed CI
2. https://github.com/zephyrproject-rtos/hal_nxp/pull/211 merged. At this point, updating the NXP HAL revision in Zephyr would cause CI failures (as the RT6xx does not currently select `CONFIG_PLATFORM_SPECIFIC_INIT`)
3. Other PRs, such as https://github.com/zephyrproject-rtos/hal_nxp/pull/217 merged
4. The associated Zephyr PRs, such as https://github.com/zephyrproject-rtos/zephyr/pull/53444 became blocked by the merge of  https://github.com/zephyrproject-rtos/zephyr/pull/52730.

To summarize, there was a dependency between https://github.com/zephyrproject-rtos/zephyr/pull/52730 and https://github.com/zephyrproject-rtos/hal_nxp/pull/211 that was not obvious until the HAL PR merged.